docs/source/usecases/customrule.rst

```diff
@@ -0,0 +1,52 @@
+Config - Custom Rules
```

Comment:
Done in f6e1be00fb4bd1abeba5b0caf8162624621d5e7e and 99b5d0a442447b1a2371eb1f11fc4a04774ba739.